### PR TITLE
Added !fish and !cheems

### PR DESCRIPTION
**Pull request by bastiengly**
!fish and !cheems added
although !cheems currently in limbo to avoid chat spam while we figure out timeouts

### CODE REVIEW

File: commands/cheems.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+};
+
+
+const embedMessage = (msg, args) => {
+  let roll = generateRoll();
+  return msg.channel.send(`${cheems[roll]}`);
+}
+
+module.exports = {
+  name: "!cheems487484854874",
```
Comment (VuongRon) [inline commands/cheems.js]: alkWut

File: commands/fish.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  }else if (roll>=90) {
+    return "Pog!"
+  }else if (roll>=75) {
+    return "nice"
+  }else if (roll>=50) {
+    return "It's brand new"
+  }else if (roll>=25) {
+    return "Maybe someone else can use it"
+  }
+}
```
Comment (VuongRon) [inline commands/fish.js]: I'd use an object here and combine them like:

``` 
return {
  "name": "You caught a :boot:!",
  "value": "It's brand new"
};
```

to keep DRY. And you can use it in `embedMessage` directly, and merge it with `{ "inline": true }`. Saves a bunch of lines.

**Review by VuongRon — COMMENTED** [review]
W o w ! 🎉 

Just a few comments for optimization purposes.

**Comment by VuongRon** [comment]
If cheems is gonna be on hold until timeouts are figured out, I recommend removing from this PR and opening a 2nd one.

**Comment by bastiengly** [comment]
Cheems fixed and fish optimized, should be ok to merge.

**Comment by VuongRon** [comment]
Now we're poggin.

**Pull request merged**